tasks.php

```diff
@@ -577,9 +577,14 @@ function create_task_from_form_submission($formsub)
                 ShowNotification($errmsg, true);
                 break;
             }
-            TaskHeader("All Open Tasks");
-            list_all_open_tasks();
-            break;
+            else
+            {
+                // If we successfully create the task, we should reload
+                //   the page to clear the POST data and make sure that
+                //   reloading does not lead to duplicated tasks.
+                header("Location: $code_url/tasks.php");
```

Comment:
The "DP" way to do this is `metarefresh()` which handles some test conditions too.

Reply:
Switched to `metarefresh` in 44ec740ad8c7fdf99dee6a4f0f5bbe066c6194c5 and 8a53710d6c0fc4d8f798834b13a96575251b00b8.